cms/envs/production.py

```diff
@@ -297,6 +297,10 @@
             CAS_ATTRIBUTE_CALLBACK['function']
         )
 
+# Login using the LMS as the identity provider
```

Comment:
Need to update this comment to say "Do NOT login using the LMS as an Identity Provider if disabled".